docs/content/what-are-the-differences-between-temporal-and-a-traditional-system.md

```diff
@@ -0,0 +1,64 @@
+---
+id: what-are-the-differences-between-temporal-and-a-traditional-system
+title: What are the differences between Temporal and a traditional system?
+description: This page compares a function execution in a traditional system and a Workflow Execution in Temporal.
+tags:
+  - explanation
+---
+
+import DetermineHeader from '../components/DetermineHeader.js'
+
+export const headingText = 'What are the differences between Temporal and a traditional system?'
+
+<DetermineHeader
+hLevel={props.heading}
+hText={headingText}
+/>
+
+The Temporal platform exists to facilitate Workflow Executions.
+This page compares a function execution in a traditional system and a Workflow Execution in Temporal.
+
+![Diagram that shows traditional function execution and Temporal Workflow Execution](/img/docs/function-execution-vs-reentrant-process.png)
+
+:::note Legend
+
+λ = traditional function execution  
+π = Temporal Workflow Execution  
+𝝈 = state
+
+:::
+
+Although the two systems seem similar at first glance, they differ in several significant ways.
+
+## Failure
+
+In a traditional system, a service function execution (λ) is both volatile and short-lived.
+
+- If a function execution fails, it's not resumable because all execution state is lost. The longer a function execution awaits, the higher the change of failure.
+- A traditional function execution typically has a limited lifespan, often measured in minutes.
+
+In Temporal, a Workflow Execution (π) is resumable.
+
+- A Workflow Execution is fully resumable after a failure.
+- Temporal imposes no deadlines on Workflow Executions.
+
+## State
+
+In a traditional system, stoppage or failure means that all execution state is lost.
+Your application (or a supporting component) must monitor the service's response to initiate a retry of the service execution.
+A retry starts from its _initial_ state.
+
+In Temporal, computation resumes from its _latest_ state. All progress is retained.
+
+## Communication
+
+In a traditional system, you can't "communicate" with a function execution.
+
+In Temporal, you can use Signals to send data into and get data out of a Workflow Execution.
```

Comment:
```suggestion
In a traditional system, passing data to and retrieving the state of a function execution is not really possible.

In Temporal, Signals and Queries enable data to be sent into or extracted from an open Workflow Execution.
```

Hmm... maybe something like this? "Communicate" feels off.

Reply:
Good call. I'd still like to avoid passive voice, but it's sometimes hard to avoid in technical docs.

Reply:
Re: "in" vs. "with". I could make a case for "in" because a service function execution occurs _in_ a traditional system (and a Workflow Function Execution _in_ Temporal). If, though, we're effectively comparing the high-level concept of a function execution in both systems, using "with" makes sense.

Reply:
"in" vs "with" - At that moment when I posted "with" sounded more appropriate...
But I actually can go either way. So, don't feel obligated to keep that change.